src/carousel.tsx

```diff
@@ -33,6 +33,7 @@ const useBulletsStyles = createUseStyles((theme) => ({
         height: 8,
         borderRadius: '50%',
         transition: 'transform 0.3s ease-in-out, background-color 0.3s ease-in-out',
+        zIndex: 2,
```

Comment:
changes in this file, zIndex 1 -> 2 are needed because now images has a zIndex 1, otherwise arrows and bullets are hidden bellow the image, any other idea?

Reply:
please, leave a comment explaining the reason of this